lang/cs/Org.Apache.REEF.Wake/Remote/IRemoteManager.cs

```diff
@@ -31,6 +31,10 @@ public interface IRemoteManager<T> : IStage
 
         IObserver<T> GetRemoteObserver(IPEndPoint remoteEndpoint);
 
+        IRemoteObserver<T> GetUnmanagedRemoteObserver(RemoteEventEndPoint<T> dest);
```

Comment:
This might break backward compatibility if a user chooses to implement `IRemoteManager`. @markusweimer do you think this is a concern?